starter-code/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "starter-code",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
add the following line to run your server with nodemon:
"start": "nodemon app.js"
(in the terminal:)
$ npm run start